pandas/tests/frame/methods/test_combine_first.py

```diff
@@ -79,9 +76,7 @@ def test_combine_first(self, float_frame, using_infer_string):
         tm.assert_series_equal(combined["A"].reindex(g.index), g["A"])
 
         # corner cases
-        warning = FutureWarning if using_infer_string else None
-        with tm.assert_produces_warning(warning, match="empty entries"):
-            comb = float_frame.combine_first(DataFrame())
+        comb = float_frame.combine_first(DataFrame())
```

Comment:
so this now does not issue a FutureWarning on main since #58056.

I guess the changes to this file would not be backported to 2.3.x. There is no xfail for this test on 2.3.x and the `FutureWarning: The behavior of array concatenation with empty entries is deprecated.` is issued when `future.infer_string = True`

Reply:
and I think you can also remove the _using_infer_string_ fixture from the function arguments too.

Reply:
Noted for backport - nice catch